aten/src/ATen/native/native_functions.yaml

```diff
@@ -4906,7 +4906,7 @@
   use_c10_dispatcher: full
   variants: method, function
   dispatch:
-    CPU: legacy::cpu::_th_dist
+    CPU: dist
```

Comment:
Is the implementation in TH used by anything else? If no, could you delete it please?

Reply:
Has deleted TH dist cpu implementation